src/writePackageJSON.ts

```diff
@@ -13,5 +13,5 @@ export async function writePackageJSON(contents: PackageJson, filePath: string)
   const ogPackageJson = await fs.readFile(p, 'utf-8');
 
   const indent = detectIndent(ogPackageJson);
-  await fs.writeFile(p, JSON.stringify(contents, null, indent.indent ?? '  '), 'utf-8');
+  await fs.writeFile(p, JSON.stringify(contents, null, indent.indent ?? '  ') + '\n', 'utf-8');
```

Comment:
for this to be cross-platform, this would need to change to utilize the `node:os` package and use the `EOL` constant that is available in there:

```javascript
import os from 'node:os';

await fs.writeFile(p, `${JSON.stringify(contents, null, indent.indent ?? ' ')${os.EOL}`, 'utf-8');
```

the sample, above, also uses the preferred [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) approach to string concatenation.

Reply:
Just pushed a commit to align with your proposal! Thanks!